### DISCUSSION TITLE
Welcome to GlassyUI-Components Discussions!

### DISCUSSION THREAD

**Discussion by Jaishree2310** [Jaishree2310/GlassyUI-Components#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jaishree2310** [comment]
@kodeCraze Before submitting any PR please ask for assigning the issue because it becomes difficult for me as issues are not assigned and PR is raised. It should be like the one who gives PR first that should be merged if it's correct.
 Thank you for your contributions!

**Reply by kodeCraze** [reply]
> @kodeCraze Before submitting any PR please ask for assigning the issue because it becomes difficult for me as issues are not assigned and PR is raised. It should be like the one who gives PR first that should be merged if it's correct. Thank you for your contributions!

@Jaishree2310,

Thank you for your feedback regarding the pull request (PR) process. I appreciate you taking the time to clarify the preferred workflow.

I understand the importance of properly assigning issues before submitting PRs. Moving forward, I will ensure to request issue assignment prior to raising any PR. This will help streamline the process and avoid any confusion. I also acknowledge your point about merging PRs on a first-come, first-served basis, provided they are correct. This approach seems fair and encourages timely contributions.

Thank you for providing these guidelines. 

For sure i look forward to adhering to these best practices in my future submissions.

Best regards,
@kodeCraze

**Comment by Jaishree2310** [comment]
@AswaniBolisetti @Sawan-Kushwah @Gauravtb2253 @gurliv21 @mohitahlawat2001 @stevenanthonyrevo @rajatsinghal02 
Could you complete it within 2 days for better contribution? I am in the Loop of checking all the PR's and issues very carefully so I have to keep the consistency maintained.

**Reply by stevenanthonyrevo** [reply]
Great idea, My suggestion would be if the Github bot can close an issue after two days if it is not eligible for merge? https://docs.github.com/en/actions/use-cases-and-examples/project-management/closing-inactive-issues
